src/Maintenance/Maintenance/ChangeLog.md

```diff
@@ -20,6 +20,8 @@
 
 ## Upcoming Release
 * Fixed bug where rebootSettings property wasn't updating.
+* Fixed bug where Get-MaintenanceConfig was returning a List.
+    - This bug was affecting Update-MaintenanceConfig's input params.
```

Comment:
```suggestion
    - This bug was affecting Update-MaintenanceConfig's input params.
```
This bug is not only affecting Update-MaintenanceConfiguration, but also affecting all cmdlets receive the return objects from Get-AzMaintenanceConfiguration like Remove-AzMaintenanceConfiguration